apps/web/components/sections/project/media-player.tsx

```
@@ -0,0 +1,83 @@
+            src={activeItem.src}
+            alt={activeItem.alt || "Media item"}
+            className="h-full object-contain"
+          />
+        ) : (
+          <video
+            src={activeItem.src}
+            controls
+            className="h-full object-contain"
+          >
+            Your browser does not support the video tag.
+          </video>
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)